keyboards/boardsource/unicorne/keymaps/vial/vial.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Unicorne",
+  "vendorId": "0x4273",
+  "productId": "0x7563",
+  "keycodes": ["qmk_lighting"],
+  "menus": ["qmk_audio", "qmk_rgb_matrix"],
```

```suggestion
  "lighting": "vialrgb",
```
None of these other settings are used by Vial.